docs/source/index.rst

```diff
@@ -1,13 +1,25 @@
-.. surround documentation master file, created by
-   sphinx-quickstart on Wed Mar  6 16:54:44 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. image:: temp_logo_hq.png
+    :alt: Surround
+    :align: center
+    :scale: 45%
 
-Welcome to surround's documentation!
+Welcome to Surround
 ====================================
 
+Welcome to Surround's documentation. For a quick rundown on getting started with Surround see :ref:`getting-started`.
```

Comment:
We need a one line description of what Surround is `Surround is a framework for building machine learning pipelines in Python`. Perhaps that for now.